ginga/rv/Control.py

```diff
@@ -281,6 +281,36 @@ def stop_local_plugin(self, chname, opname):
         opmon = channel.opmon
         opmon.deactivate(opname)
 
+    def call_local_plugin_method(self, chname, plugin_name, method_name,
+                                 args, kwargs):
+        """
+        Parameters
+        ----------
+        chname : str
+            The name of the channel containing the plugin.
+
+        plugin_name : str
+            The name of the local plugin containing the method to call.
+
+        method_name : str
+            The name of the method to call.
+
+        args : list or tuple
+            The positional arguments to the method
+
+        kwargs : dict
+            The keyword arguments to the method
+
+        Returns
+        -------
+        result : return value from calling the method
```

Comment:
Is this a `bool`, `str`, `obj`, ...?

Reply:
It depends what the method you are calling returns. What is the docstring convention for specifying any valid python type?

Reply:
I am not sure if there is one for *any*

Reply:
Ok, so maybe the current docstring is ok? ... :grin: